setup.py

```diff
@@ -8,103 +8,88 @@
+
     def run(self):
         if VERSION == DEFAULT_VERSION:
-            raise Exception(f'$WTM_VERSION was not specified in environment. Aborting.')
+            raise Exception(f"$WTM_VERSION was not specified in environment. Aborting.")
 
         # Only run `sdist` if we found a valid version in environment.
         sdist.run(self)
 
+
 setup(
     name=PKGNAME,
     version=VERSION,
-    description='Learn about tracking technologies, market structure and data-sharing on the web',
+    description="Learn about tracking technologies, market structure and data-sharing on the web",
     long_description=LONG_DESCRIPTION,
     classifiers=[
-        'Development Status :: 4 - Beta',
-        'Environment :: Console',
-        'Intended Audience :: Developers',
-        'Intended Audience :: Information Technology',
-        'Intended Audience :: Science/Research',
-        'License :: OSI Approved :: MIT License',
-        'Operating System :: OS Independent',
-        'Programming Language :: Python :: 3.6',
-        'Topic :: Internet',
-        'Topic :: Scientific/Engineering :: Information Analysis',
-    ],
-    keywords='tracking whotracksme whotracks.me who tracks',
-    url='https://whotracks.me',
-    author='Cliqz Gmbh',
-    author_email='privacy@cliqz.com',
-    license='MIT',
-    packages=find_packages(exclude=[
-        '_site',
-        'blog',
-        'contrib',
-        'data',
-        'static',
-        'templates',
-        'tests',
-    ]),
-    install_requires=[
-        'docopt',
-        'pandas',
-        'requests',
+        "Development Status :: 4 - Beta",
+        "Environment :: Console",
+        "Intended Audience :: Developers",
+        "Intended Audience :: Information Technology",
+        "Intended Audience :: Science/Research",
+        "License :: OSI Approved :: MIT License",
+        "Operating System :: OS Independent",
+        "Programming Language :: Python :: 3.6",
+        "Topic :: Internet",
+        "Topic :: Scientific/Engineering :: Information Analysis",
     ],
+    keywords="tracking whotracksme whotracks.me who tracks",
+    url="https://whotracks.me",
+    author="Cliqz Gmbh",
+    author_email="privacy@cliqz.com",
+    license="MIT",
+    packages=find_packages(
+        exclude=["_site", "blog", "contrib", "data", "static", "templates", "tests"]
+    ),
+    install_requires=["docopt", "pandas", "requests"],
     extras_require={
-        'dev': [
-            'aiohttp',
-            'boto3',
-            'colour',
-            'jinja2',
-            'markdown',
-            'plotly',
-            'pytest',
-            'sanic',
-            'squarify',
-            'twine',
-            'watchdog',
-            'libsass',
-            'pyyaml>=4.2b1',
-        ],
-    },
-    package_data={
-        f'{PKGNAME}.data': assets,
+        "dev": [
+            "aiohttp",
+            "boto3",
+            "colour",
+            "jinja2>=2.10.1",
```

Comment:
Is this needed? `requirements-dev.txt` should be enough right?

Reply:
I don't really have a solid understanding of the packaging that setup.py provides. There was an occurrence of a dependency a few lines below: `pyyaml>=4.2b1` which had it the version constraint( I assume for the same reasons), so I put it in for jinja2 as well.

Reply:
I think the difference with `pyyaml` was that the security fix required a beta version of the package, which would not get installed by default when running `pip install pyyaml` for example. In the case of `Jinja2` not specifying a version should default to the latest release.